config/locales/provider_interface/make_offer.yml

```diff
@@ -75,6 +75,8 @@ en:
               blank: Select course
             provider_id:
               blank: Select provider
+            base:
+              exceeded_max_conditions: Conditions must be %{count} or less
```

Comment:
This reads a bit odd. Could it be something like `Offer must have %{count} conditions or fewer`